debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -124,7 +125,7 @@ public void shouldReceiveChangesForInsertsWithPostgisTypes() throws Exception {
         TestHelper.executeDDL("postgis_create_tables.ddl");
         consumer = testConsumer(1, "public"); // spatial_ref_sys produces a tonne of records in the postgis schema
         consumer.setIgnoreExtraRecords(true);
-        recordsProducer.start(consumer);
+        recordsProducer.start(consumer, e -> {});
```

Comment:
Should not this line an subsequent ones also use `blackHole`?

Reply:
Ah yes, good catch. I had started with blackhole, than rebased and forgot about that for those tests which had been added in the meantime. I'll adjust.